check-pull-request-status.py

```diff
@@ -23,13 +25,21 @@
 pull_request = client.pull_request(owner='openshift', repository='openshift-ansible', number=pull_request_id)
 pull_request_statuses = client._json(client._get(pull_request.statuses_url.replace('statuses', 'status')), 200)['statuses']
 
-blocking_statuses = [
-    'aos-ci-jenkins/OS_3.5_NOT_containerized',
-    'aos-ci-jenkins/OS_3.5_NOT_containerized_e2e_tests',
-    'aos-ci-jenkins/OS_3.5_containerized',
-    'aos-ci-jenkins/OS_3.5_containerized_e2e_tests',
-    'continuous-integration/travis-ci/pr'
-]
+blocking_statuses = []
+
+with open(status_config_file) as status_config:
+    config = load(status_config)
+    versions = []
+    if target_branch == "master":
+        versions.extend(config["master"])
+    elif target_branch.startswith("release-"):
+        versions.append(target_branch)
+    else:
+        print("[ERROR] Can't determine set of statuses that shall be checked for the '{}' branch.".format(target_branch))
```

Comment:
I can see why this approach may save some space in the YAML config, but I think it may be better to just have an entirely simple mapping:

```yml
master:
  - test1
  - test2
otherbranch:
  - test1
  - test3
```

We will change the file very rarely and from that I think it's fine for us to repeat the common stuff as much as necessary. Also, when we get branches like `staging` and `staging-sprint123` this current code will break but if it's just a simple mapping we can add all the config we need.